ssh_filter_btrbk.sh

```diff
@@ -40,14 +40,14 @@ allow_exact_cmd()
 reject_and_die()
 {
     local reason=$1
-    log_cmd "auth.err" "btrbk REJECT" "$reason"
```

Comment:
I'd prefer if we could do it at least for this particular script differently. The reason simply is, that double quotes do allow expansions, which may then accidentally happen. And since this is strongly security focused, I think it's better to have expansions only happen, when we really want them

Reply:
I assume you're ok with that part since you took it over in your branch and we can thus resolve this one?

Reply:
yes I'm fine with that, I merely wanted to give an "excuse" on why I do it otherwise